library/spi_engine/interfaces/Makefile

```diff
@@ -0,0 +1,40 @@
+####################################################################################
+####################################################################################
+## Copyright (c) 2024 Analog Devices, Inc.
```

Comment:
Maybe this should follow the pattern of the other Makefiles, like the one from library/interfaces/Makefile.
So with 2018-2024 range since these are autogenerated.